css-color/Overview.bs

```diff
@@ -1872,18 +1890,8 @@ Predefined colorspaces: ''srgb'', ''dci-p3'', and ''rec2020''.</h3>
 				<tr><th>Green chromaticity</th><td>x=0.170 y=0.797</td></tr>
 				<tr><th>Blue chromaticity</th><td>x=0.131 y=0.046</td></tr>
 				<tr><th>White chromaticity</th><td>x=0.3127 y=0.3290 (D65)</td></tr>
-				<tr><th>Transfer function</th><td>see below</td></tr>
+				<tr><th>Transfer function</th><td>1/2.4</td></tr>
```

Comment:
FYI. See [ITU Rec.2087](https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.2087-0-201510-I!!PDF-E.pdf) for recommendations on "Colour conversion from Recommendation ITU-R BT.709 to Recommendation ITU-R BT.2020"